txscript/script.go

```diff
@@ -458,3 +458,15 @@ func GetPreciseSigOpCount(scriptSig, scriptPubKey []byte, bip16 bool) int {
 	shPops, _ := parseScript(shScript)
 	return getSigOpCount(shPops, true)
 }
+
+// IsUnspendable returns whether the passed scriptPubKey is unspendable, or
+// guaranteed to fail at execution.  This allows inputs to be pruned instantly
+// when entering the UTXO set.
+func IsUnspendable(scriptPubKey []byte) bool {
```

Comment:
Please make the variable name match the rest of the code base.  In particular, it's referred to as `pkScript`.